### ISSUE TITLE
rsync -a causes input/output error

### ISSUE THREAD

**Issue by greerso** [kahing/goofys#41]
```ubuntu@ip-10-0-0-27:/mnt$ rsync -a /mnt/filestore.ebs/ /mnt/cpirepsarchive.s3/```
```rsync: failed to set times on "/mnt/cpirepsarchive.s3/.": Operation not permitted (1)```
And that same error for every file. 

Then the mount looks like this:
`d?????????  ? ?        ?           ?            ? cpirepsarchive.s3/`

**Comment by kahing** [comment]
goofys doesn't support setting mtime, but since we never keep track of it, maybe we should just ignore the operation

**Comment by kahing** [comment]
actually looks like that's related to xattr, which jacobsa/fuse doesn't implement

**Comment by kahing** [comment]
Did more research into this. The problem because of chmod:

```
[pid 20039] chmod(".README.md.cwIhpA", 0664) = -1 EPERM (Operation not permitted)
```

Note that goofys itself never returned that error, and it seems to be generated somewhere within the kernel[1], [2]

[1]: https://bugzilla.samba.org/show_bug.cgi?id=5520
[2]: http://www.gossamer-threads.com/lists/linux/kernel/927626

The underly problem is that goofys does not keep track of file permissions, but `rsync -a` requests that permissions to be preserved. The kernel seems to notice that chmod is not working, and returned EPERM to rsync.

I can workaround that by faking the permission, but that can cause unexpected behaviors. Let me sit on the diff some more.

**Comment by kahing** [comment]
closed via c17aba6a4071372fdd99ea6e3654747dc61403ce
